ReactiveCocoa/Swift/Signal.swift

```diff
@@ -883,6 +890,18 @@ public func combineLatest<A, B, C, D, E, F, G, H, I, J, Error>(a: Signal<A, Erro
 		|> map(repack)
 }
 
+/// Combines the values of all the given signals, in the manner described by
+/// `combineLatestWith`.
```

Comment:
Can you please document that this operator will never send events if the sequence is empty?